externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala

```diff
@@ -76,6 +75,8 @@ object SparkSQLEngine extends Logging {
 
   private val countDownLatch = new CountDownLatch(1)
 
+  val replOutputDir = Utils.createTempDir(namePrefix = "repl").toFile.getAbsolutePath
```

Comment:
It will always create a temp dir in `java.io.tmpDir`, shall we do as same as what spark does?

```
val rootDir = conf.getOption("spark.repl.classdir").getOrElse(Utils.getLocalDir(conf))
  val outputDir = Utils.createTempDir(root = rootDir, namePrefix = "repl")
```

Reply:
org.apache.spark.util.Utils is only accessible in package `org.apache.spark`